docs/guides/extracting-data.md

```diff
@@ -2,7 +2,7 @@
 title: Extracting Data
 ---
 
-Extracting data means reading tabular data from some source. We can use various customizations for this process such as providing a file format, table schema, limiting fields or rows amounts, and much more. Let's see this with some real files:
+Extracting data means reading tabular data from a source. We can use various customizations for this process such as providing a file format, table schema, limiting field or row values, and much more. Let's see this with some real files:
```

Comment:
I might be wrong, but I think the text means limiting which fields or how many rows are to be extracted, not really their values.